recipes/c4core/all/conanfile.py

```diff
@@ -80,9 +81,9 @@ def package(self):
         self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
         cmake = self._configure_cmake()
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "cmake"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "include"), "*.natvis")
+        tools.files.rmdir(self, os.path.join(self.package_folder, "cmake"))
+        tools.files.rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        tools.files.rm(self, os.path.join(self.package_folder, "include"), "*.natvis")
```

Comment:
@AndreyMlashkin It's inverted. The pattern should come first, before the path.

Reply:
Anyway, this script rocks!

Reply:
ok, thanks. I will fix my script and re-interate